packages/strapi-design-system/.storybook/hooks/useThemeType.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState, useEffect } from 'react';
+import { parse } from 'qs';
+import addons from '@storybook/addons';
+
+const channel = addons.getChannel()
+
+export const useThemeType = () => {
+  const [isDark, setDark] = useState(false);
```

Comment:
Can it be sync with a local storage?

Reply:
Yes. we can do that in a second iteration 👌

Reply:
nice idea indeed! 
will add a card in growth backlog to not forget it 👌